### PR TITLE
docs: installers renaming changes

### PR DESCRIPTION
**Pull request by marcellorinaldo**
Documentation for PR https://github.com/eclipse-kura/kura/pull/5609

**Related Issue:** https://github.com/eclipse-kura/kura/pull/5609

**Description of the solution adopted:** N/A.

**Screenshots:** N/A.

**Manual Tests**: N/A.

**Any side note on the changes made:** N/A.

### CODE REVIEW

File: docs/getting-started/install-kura.md [hunk 1]
```diff
@@ -26,12 +26,13 @@ Installers of type (2) with the suffix `nn` are **No Networking** profiles that
 A user can deploy Kura on a target system using the installer tailored for the device architecture. The installer file looks like:
 
 ```
-kura-<kura-version>_generic-<arch>_installer.deb/rpm
+kura-<kura-version>_<arch>_installer.deb/rpm
 ```
 
-where `<arch>` is one of the **supported architectures**: *x86_64*, *arm32*, and *arm64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface.
+where `<arch>` is one of the **supported architectures**: *x86_64*, *armhf*, and *aarch64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface. Java 17 is required.
```
Comment (mattdibi) [inline docs/getting-started/install-kura.md]: I would add something more descriptive like:

```suggestion
where `<arch>` is one of the **supported architectures**: *x86_64*, *armhf*, and *aarch64*. Kura can work on systems that have available the dependencies listed in the [Kura dependencies](#kura-dependencies) section, and that have **at least one** physical ethernet interface. 

From Kura version 6.0.0 the Java 17 runtime is **required** to run the framework correctly.
```

**Review by mattdibi — COMMENTED** [review]
One final thing: since we're switching to Java17 runtime only, we can remove the "warning" callout about Java8 in the [remote debugging section](https://eclipse-kura.github.io/kura/docs-release-5.6/java-application-development/remote-debugging-on-target-platform/)

File: docs/getting-started/install-kura.md (hunk 1 above)
Comment (marcellorinaldo) [reply docs/getting-started/install-kura.md]: Addressed in [1babba4](https://github.com/eclipse-kura/kura/pull/5613/commits/1babba4085e7c85acb08879e5db05b594fb1bb4c)

File: docs/getting-started/install-kura.md [hunk 2]
```diff
@@ -26,10 +26,12 @@ Installers of type (2) with the suffix `nn` are **No Networking** profiles that
 A user can deploy Kura on a target system using the installer tailored for the device architecture. The installer file looks like:
 
 ```
-kura-<kura-version>_<arch>_installer.deb/rpm
+kura_<kura-version>_<arch>.deb/rpm
```
Comment (mattdibi) [inline docs/getting-started/install-kura.md]: No rpm on Kura

```suggestion
kura_<kura-version>_<arch>.deb
```

File: docs/getting-started/raspberry-pi-raspberryos-quick-start.md [hunk 3]
```diff
@@ -63,15 +63,15 @@ following steps:
...
+    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_<arch>_installer.deb
     ```
 
     Note: replace `<version>` in the URL above with the version number of the latest release (e.g. 5.5.0) and `<arch>` with your device architecture 
 
 5. Install Kura with: 
 
     ```
-    sudo apt-get install ./kura-<kura-version>_generic-<arch>_installer.deb
+    sudo apt-get install ./kura-<kura-version>_<arch>_installer.deb
```
Comment (mattdibi) [inline docs/getting-started/raspberry-pi-raspberryos-quick-start.md]: Remove `_installer`

```suggestion
    sudo apt-get install ./kura-<kura-version>_<arch>.deb
```

File: docs/getting-started/raspberry-pi-ubuntu-20-quick-start.md [hunk 4]
```diff
@@ -59,7 +59,7 @@ following steps:
 4. Download the Kura package with:
 
     ```
-    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_generic-<arch>_installer.deb
+    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_<arch>_installer.deb
```
Comment (mattdibi) [inline docs/getting-started/raspberry-pi-ubuntu-20-quick-start.md]: Remove `_installer`

```suggestion
    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_<arch>.deb
```

File: docs/getting-started/raspberry-pi-raspberryos-quick-start.md [hunk 5]
```diff
@@ -63,15 +63,15 @@ following steps:
 4. Download the Kura package with:
 
     ```
-    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_generic-<arch>_installer.deb
+    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_<arch>_installer.deb
```
Comment (mattdibi) [inline docs/getting-started/raspberry-pi-raspberryos-quick-start.md]: Here too

```suggestion
    wget http://download.eclipse.org/kura/releases/<version>/kura-<kura-version>_<arch>.deb
```

File: docs/getting-started/install-kura.md [hunk 6]
```diff
@@ -8,8 +8,8 @@ Eclipse Kura&trade; is provided using a Debian Linux package. Visit the [Kura do
 
 Several installers can be found on such page, and they fall into one of the following categories:
 
-1. standard installers, like `kura-6.0.0_generic-arm32_installer.deb`; and
-2. installers with suffix **nn**, like `kura-6.0.0_generic-arm32-nn_installer.deb`
+1. standard **kura** installers, like `kura_6.0.0_arm64.deb`; and
```
Comment (mattdibi) [inline docs/getting-started/install-kura.md]: I would remove the `; and` in the list... it looks like a typo

```suggestion
1. standard **kura** installers, like `kura_6.0.0_arm64.deb`
```

**Review by mattdibi — DISMISSED** [review]

**Review by mattdibi — APPROVED** [review]

**Pull request merged**